modules/standard/HDFS.chpl

```diff
@@ -516,7 +516,7 @@ proc hdfs_chapel_connect(path:string, port: int): hdfsChapelFileSystem_local{
 
   var err: syserr;
   var ret = hdfs_chapel_connect(err, path, port);
-  if err then ioerror(err, "Unable to connect to HDFS", path);
+  if err then ioerror(err, "Unable to connect to HDFS", path.c_str());
```

Comment:
This should say path.localize().c_str()
(not your fault, API changed while you were developing this PR)